qcelemental/molutil/align.py

```diff
@@ -554,34 +554,41 @@ def kabsch_quaternion(P, Q):
     return U
 
 
-def compute_scramble(nat, do_resort=True, do_shift=True, do_rotate=True, deflection=1.0, do_mirror=False):
+def compute_scramble(
+    nat: int,
+    do_resort: Union[bool, List, np.ndarray] = True,
```

Comment:
Very minor, but tuple would also be valid since these are all technically `array-like`. Would it be better to be a `Union[bool, numpy.typing.ArrayLike]` as a more complete catch-all? Same for the other 2 here as well.

Reply:
ooh, I didn't know numpy supplied `ArrayLike` -- I had only seen `array-like` for human consumption. thanks!

Reply:
I didnt either! I knew there was something in the `collections` lib, but in looking while reviewing, I found that. Its only available from 1.20+ though, not sure if thats going to be a barrier.

Reply:
I'm trying to keep qcel fairly undemanding (py36+, numpy from years back).
The below can work, but the results (w/o, w) are harder to read and less linked. So I'll stick with List, ndarray for now. Glad to know about it, though!
```
try:
    from numpy.typing import ArrayLikeMine
except ImportError:
    ArrayLike = Union[List, np.ndarray]
```

<img width="765" alt="Screen Shot 2022-06-07 at 3 29 18 PM" src="https://user-images.githubusercontent.com/2314730/172466731-5062fcec-999e-44dc-b8dc-c800c413dfcb.png">
<img width="776" alt="Screen Shot 2022-06-07 at 3 28 14 PM" src="https://user-images.githubusercontent.com/2314730/172466735-18c32e47-5a8d-42ae-a898-de3d9a031316.png">